test/templates/config.ru

```diff
@@ -0,0 +1,70 @@
+$:.push File.dirname(__FILE__)
+require "fake_middleware"
+
+module Helper
+  def self.test_root
+    File.dirname(__FILE__)
```

Comment:
Can you use `Pathname` here as well? Maybe even just extract a constant instead of module with class methods.